shacl12-core/index.html

```diff
@@ -5507,6 +5508,185 @@ <h4>sh:property</h4>
+						<div class="def-header">TEXTUAL DEFINITION</div>
+						<div class="def-text-body" data-validator="MemberShape">
+							Let <code>$memberShape</code> be a <a>value</a> of <code>sh:memberShape</code>.
+							Each <a>value node</a> <code>v</code> must be a <a href="#syntax-rule-SHACL-list">SHACL list</a> - a <a>failure</a> MUST be produced if <code>v</code> is not a SHACL list.
+							For each member <code>m</code> of the list <code>v</code>
+							a <a>failure</a> MUST be produced if the <a>conformance checking</a> of <code>m</code> against <code>$memberShape</code> produces a <a>failure</a>.
+							Otherwise, if <code>m</code> does not <a>conform</a> to <code>$memberShape</code>,
+							there is a <a>validation result</a> with <code>m</code> as <code>sh:value</code> and information about the non-conforming member.
+						</div>
+					</div>
+					<p><em>The remainder of this section is informative.</em></p>
+					<p>
+						In the following example, all values of the property <code>ex:speakerOrder</code> must be RDF lists, and each member of those lists must be an IRI.
+						Note that <code>sh:memberShape</code> can be used in both property shapes (as shown) and node shapes (where it would apply to the focus node itself).
+					</p>
+					<aside class="example">
+						<div class="shapes-graph">
+							<div class="turtle">
+ex:AgendaShape
+	a sh:NodeShape ;
+	<span class="target-can-be-skipped">sh:targetClass ex:Agenda ;</span>
+	sh:property [
+		sh:path ex:speakerOrder ;
+		sh:memberShape [
+			sh:nodeKind sh:IRI ;
+		] ;
+		sh:maxCount 1 ;
+	] .
+							</div>
+							<div class="jsonld">
+								<pre class="jsonld">{
+	"@id": "ex:AgendaShape",
+	"@type": "sh:NodeShape",
+	"sh:property": {
+		"sh:maxCount": {
+			"@type": "xsd:integer",
+			"@value": "1"
+		},
+		"sh:memberShape": {
+			"sh:nodeKind": {
+				"@id": "sh:IRI"
+			}
+		},
+		"sh:path": {
+			"@id": "ex:speakerOrder"
+		}
+	},
+	"sh:targetClass": {
+		"@id": "ex:Agenda"
+	}
+}</pre>
+							</div>
+						</div>
+						<div class="data-graph">
+							<div class="turtle">
+ex:agenda1 a ex:Agenda ;
+	ex:speakerOrder ( ex:Alice ex:Bob ex:Charlie ) .
+
+<span class="focus-node-error">ex:agenda2</span> a ex:Agenda ;
+	ex:speakerOrder ( ex:Alice ex:Bob [ rdfs:label "Charlie" ] ) .
+							</div>
+							<div class="jsonld">
+								<pre class="jsonld">{
+	"@graph": [
+		{
+			"@id": "ex:agenda1",
+			"@type": "ex:Agenda",
+			"ex:speakerOrder": {
+				"@list": [
+					{
+						"@id": "ex:Alice"
+					},
+					{
+						"@id": "ex:Bob"
+					},
+					{
+						"@id": "ex:Charlie"
+					}
+				]
+			}
+		},
+		{
+			"@id": "ex:agenda2",
+			"@type": "ex:Agenda",
+			"ex:speakerOrder": {
+				"@list": [
+					{
+						"@id": "ex:Alice"
+					},
+					{
+						"@id": "ex:Bob"
+					},
+					{
+						"rdfs:label": "Charlie"
+					}
+				]
+			}
+		}
+	]
+}</pre>
+							</div>
+						</div>
+						<div class="results-graph">
+							<div class="turtle">
```

Comment:
ReSpec/HTML double-checking question - I see the above block needed `<div class="json"><pre class="jsonld">...`.  Turtle blocks don't need the same?  I think I see a subtle font difference in the rendered document.

Reply:
I'm following what other examples in the document are doing.

If this is problematic from a ReSpec/HTML perspective then I'd suggest opening and issue as this would need to be a change applied document wide.